### DISCUSSION TITLE
Please add real-time Augmented Reality for Jewellery

### DISCUSSION THREAD

**Discussion by Vortex853** [inspireui/support#5507] [New Feature/Ideas Requests]
Please add real-time Augmented Reality for fluxstore multi-vendor that enables consumers to virtually try on Jewellery and experience how it feels to own them before the actual purchase

**Comment by minhcasi** [comment]
Hi, may i know if you have the Jewellery 3D model on your site ?

**Comment by ahmad853** [comment]
Hi, I dont have Jewellery 3D model 
But it was requested by some Vendors in my FluxStore Multi Vendors app
